.github/workflows/release_api.yml

```diff
@@ -46,30 +46,16 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version: 19
+          node-version: 20
 
       - name: Install UI Packages
-        run: make ui-setup
+        run: make ui/setup
 
       - name: UI Build Static Files
-        run: make ui-create-static-out
+        run: make ui/create-static-out
 
       - name: UI Zip and Release
         env:
           TAG: ${{ github.event.release.tag_name }}
           GH_TOKEN: ${{ github.token }}
-        run: make ui-zip-and-release tag=$TAG
-
-  cleanup:
```

Comment:
Just checking if we still need this? Presumably the docker images will start piling up on the runners longterm?

Reply:
Good shout